Components/PalindomeChecker.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+export const PalindomeChecker = () => {
+  const [input, setInput] = useState('');
+  const [isPalindrome, setIsPalindrome] = useState(false);
+
+  function isArrayLengthEven (arrLength: number) {
+    if (arrLength % 2 === 0){
+      return true;
+    }
+      return false;
+  }
+
+  const isItAPalindrome = (): void => {
```

Comment:
This function works when I tested with a number of different cases! what edge cases could we consider?

Reply:
My first thought was we should add a check to handle spaces, but it appears a palindrome isn't limited to a single word. 

Coming up empty on any additional edge cases rn.